src/Umbraco.Cms.ManagementApi/ViewModels/ExamineManagement/SearchResultViewModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Umbraco.Cms.ManagementApi.ViewModels.ExamineManagement;
+
+public class SearchResultViewModel
+{
+    [JsonPropertyName("id")]
+    public string? Id { get; set; }
+
+    [JsonPropertyName("score")]
+    public float Score { get; set; }
+
+    [JsonPropertyName("fieldCount")]
+    public int FieldCount => Values?.Count ?? 0;
+
+    [JsonPropertyName("values")]
+    public IReadOnlyDictionary<string, IReadOnlyList<string>>? Values { get; set; }
```

Comment:
This should be remodelled as strongly typed models and maybe renamed to `Fields`. Something like:

```csharp
[JsonPropertyName("fields")]
public IEnumerable<Field> Fields { get; set; }

...

public class Field 
{
  public string Name { get; }
  
  public IEnumerable<string> Values { get; }
}

```